packages/volto/news/6121.bugfix

```diff
@@ -0,0 +1 @@
+fix(teaser): reset button undefined when no target @nileshgulia1
```

Comment:
@nileshgulia1 I think you should change the bugfix notice since Volto doesn't use
conventional commits style.

Reply:
Done in [8b9f5f9](https://github.com/plone/volto/pull/6121/commits/8b9f5f9a3965b15b65b3901757319134ac2bb3e2)